app/controllers/admin/users_controller.rb

```diff
@@ -16,7 +16,8 @@
 =end
 
 class Admin::UsersController < ApplicationController
-  
+  before_action :admin_authorization
+
   GUEST_USER_ROLE_ID = '3'
```

Comment:
Try not to have naked constants.  If referring to some constant, use a semantic constant for it.

Reply:
Hmm sorry I don't really understand the suggestion here. Should I rename admin_authorization to :only_allow_admins_to_access?